src/commands/hook/remove/index.js

```diff
@@ -1,23 +1,20 @@
 // @flow
 import fs from 'fs'
-import execa from 'execa'
 import ora from 'ora'
 
 import HOOK from '../hook'
+import getAbsoluteHooksPath from '../../../utils/getAbsoluteHooksPath'
 
 const removeHook = async () => {
   const spinner = ora('Creating the gitmoji commit hook').start()
 
   try {
-    const { stdout } = await execa('git', ['rev-parse', '--absolute-git-dir'])
+    const hookFile = await getAbsoluteHooksPath(HOOK.FILENAME)
 
-    fs.unlink(stdout + HOOK.PATH, (error) => {
-      if (error)
-        return spinner.fail('Error: Gitmoji commit hook is not created')
-      spinner.succeed('Gitmoji commit hook removed successfully')
-    })
+    fs.unlinkSync(hookFile)
+    spinner.succeed('Gitmoji commit hook removed successfully')
```

Comment:
I'm curious, what did we changed this? 🤔

Reply:
There were some `UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'fail' of undefined` in the tests, I felt easier to handle that with the `*Sync` API inside an async function